docs/topics/telemetry.md

```diff
@@ -12,15 +13,23 @@ This data will be used to collect aggregate, non-identifiable data to help us an
   - Node pool counts and VM Sku data
   - Which versions of VHDs are still in use
 
-This data will be used to monitor the health of cluster deployments (including AKS clusters) that are deployed with aks-engine as well as to help us prioritize future investments/feature work in the tool.
+This data will be used to monitor the health of cluster deployments (including AKS clusters) that 
+are deployed with aks-engine as well as to help us prioritize future investments/feature work in 
+the tool.
 
 ## Configuration
 
 Collection of all telemetry is currently **disabled** by default.
 
-Telemetry can be enabled by setting the `EnableTelemetry` feature flag to `true` in the apimodel.json.
+Telemetry can be enabled by setting the `enableTelemetry` feature flag to `true` in the 
+apimodel.json. When `enableTelemetry` is set to `true`, telemetry will be sent to the aks-engine 
+Application Insights cluster.
 
-Telemetry can be routed to different application insights instance be specifying `telemetryProfile.applicationInsightsKey` to the instrumentation key of your application insights instance.
+Telemetry can be routed to an additional Application Insights instance by specifying 
```

Comment:
Seems we should have a statement about where the data goes by default before we introduce an additional sink

Reply:
Just before this, I say the following:
> When `enableTelemetry` is set to `true`, telemetry will be sent to the aks-engine 
 Application Insights cluster.

Should this be rephrased?